### PR TITLE
Update available node versions for js-storefront from sap documentation

### PR DESCRIPTION
**Pull request by fibsifan**
Signed-off-by: Johannes Ballmann github@jball.de

### CODE REVIEW

**Comment by mlytvyn** [comment]
Thank you for your contribution!

**Comment by mlytvyn** [comment]
@fibsifan , can you please update CHANGELOG.md accordingly?

File: CHANGELOG.md [hunk 1]
```diff
@@ -44,6 +44,7 @@
 - Show once `CHANGELOG.md` in a new `What's New` tab once per version [#1125](https://github.com/epam/sap-commerce-intellij-idea-plugin/issues/1125)
 - Show bundled `What's New` tab with fallback to GitHub page [#1131](https://github.com/epam/sap-commerce-intellij-idea-plugin/pull/1131)
 - Updated gradle wrapper to 8.7 [#1126](https://github.com/epam/sap-commerce-intellij-idea-plugin/pull/1126)
+- Updated JSON-schema for js-storefront manifest.json to reflect changes from SAP documentation.
```
Comment (mlytvyn) [inline CHANGELOG.md:47]: please, add link to PR, besides that it would be beneficial to mention that change under `CCv2 enhancements` section

**Comment by fibsifan** [comment]
Done

**Pull request merged**